Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:8
+WORKDIR /app
+COPY . ./
+RUN yarn install
+RUN npm run build
+WORKDIR /app/deployment/server
+EXPOSE 8080
+ENTRYPOINT [ "npm", "run", "start" ] 
```

Comment:
HINT: Use a linter to make github happy like coming with the VS Code Extension (`Ctrl + Shift + x`)

```
Docker
Kubernetes
YAML
```